src/styles/_card.scss

```diff
@@ -0,0 +1,132 @@
+@use 'variables';
+
+.card {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: variables.$mainColor;
+
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
+  padding-inline: 16px;
+  padding-block: 32px 16px;
+
+  border: 1px solid variables.$cardBorderColor;
+
+  border-radius: 5px;
+
+  &__image {
+    display: block;
+    width: 160px;
+    height: 134px;
+
+    margin-inline: 3px;
+  }
+
+  &__title {
+    margin-top: 40px;
+  }
+
+  &__code {
+    margin-top: 4px;
+
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: variables.$productCodeColor;
```

Comment:
Ensure that `variables.$productCodeColor` is defined in the `variables` file. If it's not defined, this will cause an error during compilation.